src/pug/abstract/mixins.pug

```diff
@@ -0,0 +1,131 @@
+mixin topbar(currentPage)
+    -const topbarItems = [{"name":'Tài khoản của tôi',"href":"#"},{"name":'Trạng thái đơn hàng',"href":"#"},{"name":'Danh sách ưa thích',"href":"#"},{"name":'Đăng nhập',"href":"#"},{"name":'Đăng ký',"href":"#"}];
+    ul.topbar__list.col-lg-7.col-md-12        
+        each item in topbarItems
+            li.topbar__item
+                a("href"= item.href)= item.name
+    .topbar__search.col-lg-4.col-md-12
+        form
+            input(type="text",placeholder="Tìm kiếm ở đây")
+            button(aria-label="Search button")
+                i.bx.bx-search-alt-2
+
+mixin navigation(currentPage)
+    -const navItems = [{name:"Trang chủ",href:"#"},{name:"Rượu vang đỏ",href:"#"},{name:"Rượu trắng",href:"#"},{name:"Champange",href:"#"},{name:"Thông tin",href:"#"},{name:"Blog",href:"#"},{name:"Liên hệ",href:"#"}];
+    -const itemCol1 = {title:'Rượu ngoại',list:[{text:'Rượu chivas',href:'../pages/detail/detail.html'},{text:'Rượu Walker',href:'!#'},{text:'Rượu Quế',href:'!#'},{text:'Rượu Tequilla',href:'!#'},{text:'Rượu Treva',href:'!#'}]};
+    input#menuToggle.nav__toggle(type='checkbox')
+    label.nav__icon(for='menuToggle')
```

Comment:
thẻ `label` và `input` có thể liên kết bằng cách dùng `id` và `for` như em đang làm hoặc đơn giản hơn em có thể dùng thẻ `label` bọc thẻ `input` cũng được nhé